exporter/elasticsearchexporter/README.md

```diff
@@ -78,7 +78,20 @@ All other defaults are as defined by [confighttp].
 
 ### Queuing
 
-The Elasticsearch exporter supports the common [`sending_queue` settings][exporterhelper]. However, the sending queue is currently disabled by default.
+The Elasticsearch exporter supports the common [`sending_queue` settings][exporterhelper]. The sending queue is enabled by default.
+
+Default `num_consumers` is `100`. 
```

Comment:
Perhaps worth mentioning that this does not translate to the number of open connections to the Elasticsearch endpoint.

Ideally also give some guidelines on when users might want to change this and how, or maybe even just say that users should not need to change this value. :slightly_smiling_face: